lib/assessment.rb

```diff
@@ -13,28 +15,30 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(0) do |memo,item|
+      memo + item
+    end
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.any? {|number| number.odd? }
```

Comment:
Correct. The shorter version would be
`numbers.any?(&:odd?)` / `numbers.any?(:odd?)`